main/feed_data.R

```diff
@@ -1,11 +1,17 @@
+library(plyr)
+library(readr)
+
 df <- data.frame(
   id = c(1,2,3,4,5),
   text = c("great","not great", "you jerk", "bananas", "orange jerks"),
   label = c(F, F, T, F, NA),
   training = c(T, T, T, T, F)
 )
+getwd()
+df <- read.csv("./src/data/both_t_data.csv")
+#df <- read.csv("~/Desktop/example_data.csv")
+df <- rename(df, c("X_id" = "id"))
 
-library(readr)
-system2("bazel-bin/main/train_classifier_g",
+system2("../../main/train_classifier_g",
```

Comment:
what changes such that the ../.. is needed now? feels a bit brittle

Reply:
Because py_binary and r_binary are in two different branches in the tree:
```
--bazel-bin
  -- main
    -- feed_data (bazel binary)
    -- feed_data.runfile
       -- __main__
          -- main/
             -- feed_data.R
    -- train_classifier_f (bazel binary)
```